GEMINI.md

```
@@ -0,0 +1,43 @@
+* **Project Name**: Poti (Android)
+* **Language**: Kotlin (Strictly Typed)
+* **UI**: Jetpack Compose (Material3)
+* **Architecture**: Clean Architecture (4 Layers: Core, Data, Domain, Presentation)
+* **Pattern**: MVI (Model-View-Intent) with Unidirectional Data Flow (UDF)
+* **DI**: Hilt (`@HiltViewModel`, `@Inject`, `@Module`)
+* **Async**: Coroutines & Flow
+* **Network**: Retrofit2 + OkHttp + Kotlinx Serialization
+
+### 2. Strict Architectural Rules (Do's & Don'ts)
+* **Domain Layer (Pure)**:
+    * MUST be pure Kotlin. **NO** Android dependencies (`Context`, `Parcelable`, etc.).
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)